src/axom/core/execution/execution_space.hpp

```diff
@@ -109,4 +123,31 @@ struct execution_space
   #include "axom/core/execution/internal/hip_exec.hpp"
 #endif
 
+namespace axom
+{
+
+/// \brief Return default allocator id for a runtime policy.
+inline int policyToDefaultAllocatorID(axom::runtime_policy::Policy policy)
+{
+  return
+#if defined(AXOM_RUNTIME_POLICY_USE_OPENMP)
+    policy == axom::runtime_policy::Policy::omp
+    ? axom::execution_space<axom::OMP_EXEC>::allocatorID()
+    :
+#endif
+#if defined(AXOM_RUNTIME_POLICY_USE_CUDA)
+    policy == axom::runtime_policy::Policy::cuda
+    ? axom::execution_space<axom::CUDA_EXEC<256>>::allocatorID()
```

Comment:
I was surprised to see CUDA_EXEC<256> here as these are typically defined closer to where the algorithms are instantiated with execution spaces. I guess CUDA_EXEC<> with different numbers would return the same allocator.

Reply:
Yes, I think the allocator is independent of the block size.